db/migrate/20190218203848_remove_extra_columns_from_tagging_join_tables.rb

```diff
@@ -0,0 +1,19 @@
+class RemoveExtraColumnsFromTaggingJoinTables < ActiveRecord::Migration[5.2]
+  def change
+    remove_column :container_group_tags, :created_at, :datetime
+    remove_column :container_image_tags, :created_at, :datetime
+    remove_column :container_node_tags, :created_at, :datetime
+    remove_column :container_project_tags, :created_at, :datetime
+    remove_column :container_template_tags, :created_at, :datetime
+    remove_column :service_offering_tags, :created_at, :datetime
+    remove_column :vm_tags, :created_at, :datetime
+
+    remove_reference :container_group_tags, :tenant, :index => true, :null => false,  :foreign_key => {:on_delete => :cascade}
```

Comment:
@kbrock @Fryguy what is the requirement for the `tenant_id` should it be in every table? Were we planning to do partitioning/sharding based on this key?

Reply:
If it is just a join table and never exposed I don't think we need a tenant

Reply:
hm, removing the tenant_id is not revertable, since it's NOT_NULL